### ISSUE TITLE
Bug: Product and Product API created in the main API-file

### ISSUE THREAD

**Issue by mikaelsand** [Azure/azure-api-management-devops-resource-kit#838]
### Report

I use the extractor with these settings:

```JSON
{
    "sourceApimName": "my-apim",
    "destinationApimName": "my-other-apim",
    "resourceGroup": "my-apim-rg",
    "fileFolder": "/Dev/code/APIs/myapi/extract/apitemplates",
    "apiName": "my-api",
    "linkedTemplatesBaseUrl": "https://",
    "linkedTemplatesUrlQueryString": "yes",
    "policyXMLBaseUrl": "",
    "policyXMLSasToken": ""
}
```

All the files are created, including [my-apim]products.template.json and productsAPI.template.json but these are also added to the main "api-file [my-apim]-[my-api]-api.template.json

### Expected Behavior

The product and productAPI connection should not be in the API-file. They are defined in separate files. I might not want to deploy products between platforms.

### Actual Behavior

They are added into the "main api file".

```JSON
{
      "apiVersion": "2021-08-01",
      "type": "Microsoft.ApiManagement/service/products/apis",
      "name": "[concat(parameters('apimServiceName'), '/objectname/api-name')]",
      "dependsOn": [
        "[resourceId('Microsoft.ApiManagement/service/apis', parameters('apimServiceName'), 'api-name')]"
      ],
      "properties": {
        "displayName": "ProductName",
        "description": "Description here",
        "subscriptionRequired": true,
        "approvalRequired": true,
        "state": "published"
      }
    }
```

### Steps to Reproduce the Problem

1. Use the same config-file I provided
2. Get an API that are used in a Product.
3. Look at the generated result

### Component

Extractor

### DevOps Resource Kit Version

1.0.0

### .NET Version

6.0.402

### OS

Win10 21H2

### Configuration

Configuration:

```json
{
    "sourceApimName": "my-apim",
    "destinationApimName": "my-other-apim",
    "resourceGroup": "my-apim-rg",
    "fileFolder": "/Dev/code/APIs/myapi/extract/apitemplates",
    "apiName": "my-api",
    "linkedTemplatesBaseUrl": "https://",
    "linkedTemplatesUrlQueryString": "yes",
    "policyXMLBaseUrl": "",
    "policyXMLSasToken": ""
}
```

### API Management Configuration

Developer SKU
Yes. Full admin.
Nope, no VNET

### Logs

```shell
example
```

### Contact Details

mikael_sand@hotmail.com

**Comment by adrianhall** [comment]
This is by design to allow the complete transition of an API with all settings.   I'm not sure what the expected result is once you import?

**Comment by mikaelsand** [comment]
But... Why? The master template points to the Product and ProductAPI-files, which define the exact same thing. Why "hide" (and duplicate) them in the main API-file? 
The main APi file contains APIs, Operations, policies, schemas and individual diagnostic settings, why add Products here and not, lets say, Users?

Why add the Products here? The Extractor does not duplicate anything else. I just can't see the use for it. Instead, I have to manually update the main API-file every time I need to export.

**Comment by f-alizada** [comment]
Hello @mikaelsand, thank you for raising this. 
The issue was closed since PR #856 was merged into the main branch and the changes will be available in future releases. 
PR removes the product api resources from apis template.
Please feel free to reopen/create new issue if you will face issues after upgrading to the latest version. 
Best regards

**Comment by mikaelsand** [comment]
Hey! The system works :-)
